src/api/config/initializers/airbrake.rb

```diff
@@ -50,8 +50,12 @@
   # A list of parameters that should be filtered out of what is sent to
   # Airbrake. By default, all "password" attributes will have their contents
   # replaced.
-  # https://github.com/airbrake/airbrake-ruby#blacklist_keys
-  c.blacklist_keys = [/password/i, /authorization/i]
+  # https://github.com/airbrake/airbrake-ruby#blocklist_keys
+  if RailsVersion.is_6_1?
```

Comment:
This is supposed to be an example or is this really different? :smile:

Reply:
It's really different and it serves as an example at the same time. It's a win-win situation!